src/configuration.production.json

```diff
@@ -0,0 +1,54 @@
+{
+  "teamIds": [],
+  "groups": [],
+  "environment": "production",
+  "policy": "durable",
+  "endpoints": {
+    "api": "https://automation.atomist.com/registration",
+    "graphql": "https://automation.atomist.com/graphql/team"
+  },
+  "http": {
+    "enabled": true,
+    "host": "localhost",
+    "port": 2866,
```

Comment:
There's a lot of duplication, might it not be better to keep the default and then override just what needs overriding with an additional `loadEnvironmentConfig()`?

Reply:
That would also resolve the `null`/`undefined` issue.

Reply:
Yeah, I think that would be cleaner. Let me try that and see how we like that.